### PR TITLE
[ADD] login & register page

### PR DESCRIPTION
**Pull request by Adam31-jpg**

### CODE REVIEW

File: pages/login/index.jsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    const goRegister = () => {
+        setStateOnglets(2)
+    }
+
+    const goLogin = () => {
+        setStateOnglets(1)
+    }
+
+    function number(input) {
+        console.log(input)
```
Comment (I-K-M) [inline pages/login/index.jsx]: console log oublié

File: pages/login/index.jsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        let num = /[^0-9]/gi;
```
Comment (I-K-M) [inline pages/login/index.jsx]: remove this please

File: pages/login/index.jsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (I-K-M) [inline pages/login/index.jsx]: erase these lines please

File: pages/login/index.module.scss [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+            background: #67568c;
+          }
+        }
+        input{
+          background: transparent;
+          border: 1px solid #1f1235;
+          border-radius: 6px;
+          padding: 16px 50px 16px 3px;
+          margin-bottom: 30px;
+
```
Comment (I-K-M) [inline pages/login/index.module.scss]: erase this line please

File: pages/login/index.module.scss [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  display: block;
+}
+
+
+.formNames{
+  display: flex;
+  justify-content: space-between;
+  input{
+    width: 45%;
+
```
Comment (I-K-M) [inline pages/login/index.module.scss]: ''

File: pages/login/index.module.scss [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+
+
+
+
+
+
+
+
```
Comment (I-K-M) [inline pages/login/index.module.scss]: remove plz

**Review by I-K-M — CHANGES_REQUESTED** [review]

**Review by remi-boivin — APPROVED** [review]

**Pull request merged**